AlumniConnect/settings/development.py

```diff
@@ -2,7 +2,7 @@
 
 SECRET_KEY = "dikj)qxgkhe7v$y7l))d!!nkut&^6q7+x^@ys7c1z!#!&*94r5"
 
-DEBUG = True
+DEBUG = False
```

Comment:
```suggestion
DEBUG = False
```
Why is Debug = False? Make it true again. We'll make it false when and if we want to test.